imdb_back/users/models.py

```diff
@@ -1,3 +1,7 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+class User(AbstractUser):
+    verification_code = models.TextField(max_length=4, blank=False)
+    is_verified = models.BooleanField(default=False)
+    profile_picture = models.TextField(blank=True)
```

Comment:
```suggestion
    profile_picture = models.FileField(upload_to...., blank=True)
```
pa ce ti DJango sam hendalti sve za upload